db/migrate/20160309073823_update_follower_relationships_based_on_cloned_from.rb

```diff
@@ -69,8 +84,17 @@ def expand_person(person_id)
                 cloned_from = #{quote(person_id)}").to_ary
   end
 
-  def create_followers_query(person_id, follower_id, created_at, updated_at)
-    "INSERT INTO follower_relationships (person_id, follower_id, created_at, updated_at)
-     VALUES(#{quote(person_id)}, #{quote(follower_id)}, #{quote(created_at.to_s(:db))}, #{quote(updated_at.to_s(:db))})"
+  def create_insert_values(person_id, follower_id, created_at, updated_at)
+    "(#{quote(person_id)}, #{quote(follower_id)}, #{quote(created_at.to_s(:db))}, #{quote(updated_at.to_s(:db))})"
+  end
+
+  def create_insert_statement(insert_values, ignore)
```

Comment:
As a general comment (no need to change this now), readability of the method signature can be improved by using `opts` Hash. For example, if someone reads this:

```
create_insert_statement(batch, false)
```

...it's a difficult to know what does the last boolean argument do without looking at the method implementation. This improves it a bit:

```
create_insert_statement(batch, ignore: false)
```

In the implementation, you can then change:

```
def create_insert_statement(insert_values, ignore)
  "INSERT#{ignore ? " IGNORE " : " "}INTO ..."
end
```

to

```
def create_insert_statement(insert_values, opts = {})
  "INSERT#{opts[:ignore] ? " IGNORE " : " "}INTO ..."
end
```